### PR TITLE
Dbet 960 fix date of birth input

### PR DESCRIPTION
**Pull request by bradleySuira**

### CODE REVIEW

File: src/Account/BasicAccountInfo/SelectCountry/SelectCountryState.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export interface ISelectCountryState {
+    selectedCountry?: { value: string; label: string } | null
```
Comment (molekilla) [inline src/Account/BasicAccountInfo/SelectCountry/SelectCountryState.ts]: move out to interface

File: src/Account/BasicAccountInfo/SelectCountry/countries.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import countries from 'iso-3166-1/src/iso-3166'
+const COUNTRY_LIST: [{ value: string; label: string }] = countries.map(
```
Comment (molekilla) [inline src/Account/BasicAccountInfo/SelectCountry/countries.ts]: move out to interface or class

File: src/Account/BasicAccountInfo/SelectCountry/SelectCountryState.ts (hunk 1 above)
Comment (bradleySuira) [reply src/Account/BasicAccountInfo/SelectCountry/SelectCountryState.ts]: ok, fixed in the other PR

File: src/Account/BasicAccountInfo/SelectCountry/countries.ts (hunk 2 above)
Comment (bradleySuira) [reply src/Account/BasicAccountInfo/SelectCountry/countries.ts]: ok, fixed in the next PR

**Pull request merged**